pages/concepts/protocols/portal-sub-protocols/history.md

```diff
@@ -0,0 +1,85 @@
+# History Network
+
+***Note that this network is not yet fully production-ready***
+
+The chain history data consists of historical **block headers**, **block bodies** (transactions and ommer) and **block receipts**.
+It is expected that this network will one day allow clients that have pruned historical blocks to retrieve them on-demand from this network, and nodes may be able to use blocks downloaded from the History Network to sync, rather than maintaining a local copy of the blockchain. 
+
+The History Network is a [Kademlia](../kademlia.mdx) DHT that uses the [Portal Wire Protocol](./portal-wire-protocol.md) to establish an overlay network on top of the [Discovery v5](../discovery.mdx) protocol.
+
+The History Network also provides a [hash accumulator](../../hash-accumulators.mdx) that enables clients to verify that data is from the canonical Ethereum chain.
+
+## Data
+
+Portal clients can retrieve the following data types from the History Network:
+
+* Block headers
+* Block bodies
+    * Transactions
+    * Ommers
+* Receipts
+* Header epoch accumulators (pre-merge only)
+
+### Data Types
+
+#### Block Header
+
+Block header data can include: 
+- A `BlockHeaderWithProof`
+- A `BlockHeaderProof`
+- An `AccumulatorProof`
+
+The `BlockHeaderWithProof` is the actual block header data wrapped in a higher level structure with a Merkle proof demonstrating its inclusion in the canonical Ethereum chain. The proof is of type `BlockHeaderProof`, which is a `Union` that can be either `None` or `AccumulatorProof`. The `header` field in `BlockHeaderWithProof` contains the RLP encoded header encoded as an [SSZ](../ssz.mdx) bytelist.
+
+For pre-merge headers, clients SHOULD NOT accept headers without a proof as there is the `AccumulatorProof` solution available.
+For post-merge headers, there is currently no proof solution and clients SHOULD accept headers without a proof.
```

Comment:
```suggestion
For post-merge headers, clients SHOULD accept headers without a proof, and determine validity using the Beacon network.
```

Validating headers by beacon is a work in progress, not sure whether that's worth noting here. Hopefully that's on the scale of a couple of months, rather than 6-12 months away...